profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java

```diff
@@ -50,6 +50,7 @@ public OQLEngineTest() {
 
     @BeforeClass
     public static void setUpClass() throws Exception {
+        System.setProperty("polyglot.js.nashorn-compat", "true");
```

Comment:
This magic option will be moved to `Bindings` [once this change](https://github.com/graalvm/graaljs/commit/af68dca48fa16f7542e6f363ca8099d02a3d8ccd) gets in.